### PR TITLE
Implement auth0

### PR DESCRIPTION
**Pull request by ANGIECATHERINEP**
Implement auth0

### CODE REVIEW

**Review by novadeveloper14 — APPROVED** [review]

File: src/config/environment.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const IS_PRODUCTION = import.meta.env.PROD;
+const AUTH_REDIRECT_URI = import.meta.env.VITE_AUTH0_REDIRECT_URI;
+
+const enviroment = {
+  CLIENT_ID: import.meta.env.VITE_AUTH0_CLIENT_ID,
+  CLIENT_SECRET: import.meta.env.VITE_AUTH0_CLIENT_SECRET,
+  AUTH0_DOMAIN: import.meta.env.VITE_AUTH0_DOMAIN,
+  REDIRECT_URI: IS_PRODUCTION ? window.location.origin : AUTH_REDIRECT_URI,
+  TEMP_BUSINESS_UNIT: "test",
```
Comment (carin91) [inline src/config/environment.ts:9]: the TEMP_BUSINESS_UNIT  is "Test"

File: src/routes/login.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { ErrorNotClient } from "@pages/login/errors/ErrorNotClient";
+import { LoadingApp } from "@pages/login/outlets/LoadingApp";
+
+
+export interface IClients {
+  clients: IClient[];
+}
+function LoginRoutes() {
+  const clients = clientsDataMock;
+  console.log("routes");
```
Comment (carin91) [inline src/routes/login.tsx:19]: delete every console.log

File: src/pages/login/outlets/Clients/styles.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      margin: 48px auto 0px;
+      width: 500px;
+      @media screen and (max-width: 532px) {
+        width: auto;
+      }
+    }
+  }
+
+  & button {
+    margin-top: 24px;
```
Comment (carin91) [inline src/pages/login/outlets/Clients/styles.ts:19]: pass every margin, gap and padding with a value from inube.spacing

**Review by carin91 — APPROVED** [review]

File: src/pages/login/outlets/Clients/styles.ts (hunk 3 above)
Comment (ANGIECATHERINEP) [reply src/pages/login/outlets/Clients/styles.ts:19]: I spoke with Cesar and he told me that with the implementation of the inubekit library the spacing was removed, that is, now it must be placed in px

**Pull request merged**